snowplow/advanced-analytics-web-accelerator
# Feature/create hugo site

### Initial draft PR only: 

Some changes still expected to come:
1. Tracking Overview - "React Notes from agnes" needs to be replaced and potentially relevant content changed.
2. Additional sample data upload method to be added (if works).

### How to test:
- Please install Hugo first:
``` brew install Hugo```
- Clone the repo
- Checkout this branch
- Add submodule that does not get cloned but it is needed
```git submodule update --init --recursive```
- Run `hugo serve`
- Check the results in your browser 
http://localhost:1313/advanced-analytics-web-accelerator/
